### DISCUSSION TITLE
Implement InputMask Support for Calendar

### DISCUSSION THREAD

**Discussion by kgrellus** [primefaces/community#1324] [PrimeNG] [answered]
Current behavior

It is not possible to configure an inputMask for the Calendar Component.

Expected behavior

The Calendar component can use a mask attribute like the p:datePicker PrimeFaces Component.
https://primefaces.github.io/primefaces/13_0_0/#/components/datepicker?id=attributes -> mask

**Comment by Wandmalfarbe** [comment]
It looks like this feature will never be implemented ... see the now closed feature request in the PrimeNG issue tracker here https://github.com/primefaces/primeng/issues/1426

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

I've added in 17.Future we'll try to address it after the passthrough and unstyled mode implementations are done.
